src/sass/_utility-global.scss

```diff
@@ -0,0 +1,18 @@
+// Global utility classes.
+// NOTE Try you best not to use them. Seek a chance to destroy.
```

Comment:
I don't think there is anything wrong with having a set of global css classes like:
`ellipsis`, `cursor-pointer` etc

Reply:
@jean-moldovan 
We can provide a set of global classes to be used in app (for speed). Using them in components would be poor design choice though.